README.md

```diff
@@ -1,27 +1,35 @@
-# datadog-mobile-app-run-tests
+# Datadog Continuous Testing for Bitrise
 
-placeholder description
+<!-- TODO add link to marketplace after we publish the step -->
+<!-- [![Visual Studio Marketplace Version]()][1001]  -->
+[![Build Status](https://app.bitrise.io/app/7846c17b-8a13004fc7-aced-5f3b0b2ec6c4/status.svg?token=480MdFpG78E6kZASg5w1dw&branch=main)](https://app.bitrise.io/app/7846c17b-8a13004fc7-aced-5f3b0b2ec6c4)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# Overview
 
-## How to use this Step
+With the `synthetics-test-automation-bitrise-step-run-tests` step you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002].
 
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
+## Setup
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+To get started:
 
-Step by step:
+1. Add this step to your workflow. You can see [Bitrise's documentation][4001] on how to do that. You can also configure it locally by referencing this step in your `bitrise.yml`.
+2. Add your API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found here - [Setting a Secret][4002].
+3. Configure your step inputs ([Step Inputs][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs are described in a later section.
+
+## How to use this Step locally
```

Comment:
```suggestion
## How to use this step locally
```
It seems that "step" is lowercased most of the time, so I'm lowercasing it here for consistency, but you could also capitalize it throughout if you prefer that.

Reply:
I think this one i'll leave capitalised as `Step` is the terminology that Bitrise uses for this kind of integration